src/__tests__/mock/bridge/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { vi } from "vitest";
+import * as UtilsModule from "../../../bridge/utils";
+
+export function mockNonce() {
+  const nonceSpy = vi.spyOn(UtilsModule, "getNonce");
+  // prettier-ignore
+  // @ts-expect-error mock nonce
+  const nonceBuffer = Buffer.from(["59", "18", "4e", "21", "62", "17", "8a", "2b", "b2", "27", "1a", "97", "50", "6d", "e9", "a9", "a3", "b8", "c7", "9e", "fa", "0c", "54", "38", "9d", "97", "30", "e0", "c7", "8e", "07", "12"]);
```

Comment:
You can use Buffer.from("0102ffce...", "hex")

Reply:
I made changes addressing the comment.